students/DiannaTingg/lessons/lesson02/assignment/code/charges_calc.py

```diff
@@ -0,0 +1,131 @@
+"""
+Returns total price paid for individual rentals
+"""
+
+import argparse
+import json
+import datetime
+import math
+import logging
+
+LOG_FORMAT = "%(asctime)s %(filename)s:%(lineno)-3d %(levelname)s %(message)s"
+FORMATTER = logging.Formatter(LOG_FORMAT)
+LOGGER = logging.getLogger()
+
+
+def parse_cmd_arguments():
+    """
+    Parses arguments from command line
+    :return:
+    """
+    parser = argparse.ArgumentParser(description='Process some integers.')
+    parser.add_argument('-i', '--input', help='input JSON file', required=True)
+    parser.add_argument('-o', '--output', help='output JSON file', required=True)
+    parser.add_argument('-d', '--debug', help='enable debug logging', required=False, default=0)
+
+    return parser.parse_args()
+
+
+def set_log_level(log_level):
+    """
+    Sets log level for debugger
+    :param log_level: 0, 1, 2, or 3
+    :return:
+    """
+
+    if log_level == 0:
+        return
```

Comment:
Looks good. Note, you could even move the other logging stuff from the top of the file under this return. Even less overhead when not logging.